code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -550,6 +550,28 @@
 	required_reagents = list("mindbreaker" = 1, "acetone" = 1, "inaprovaline" = 1)
 	result_amount = 3
 
+/datum/chemical_reaction/cardox
+	name = "Cardox"
+	id = "cardox"
+	result = "cardox"
+	required_reagents = list("platinum" = 1, "carbon" = 1, "sterilzine" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/cardox_removal

```

Comment:
See Lohikar's comments about the ammonia - cardox can't be a cleaner if it's absorbed in the reaction, it needs to be a catalyst.

Reply:
I was under the impression that cleaners lose a bit of their strength when used, hence the 0.1 unit removal.